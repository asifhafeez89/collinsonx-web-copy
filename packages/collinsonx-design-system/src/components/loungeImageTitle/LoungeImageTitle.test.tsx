import renderer from 'react-test-renderer';
import LoungeImageTitle from '.';

describe('<LoungeImageTitle />', () => {
  it('renders LoungeImageTitle', () => {
    const tree = renderer
      .create(
        <LoungeImageTitle
          price="80"
          location="Athens, Greece"
          image="image.png"
          title="LoungeX"
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
