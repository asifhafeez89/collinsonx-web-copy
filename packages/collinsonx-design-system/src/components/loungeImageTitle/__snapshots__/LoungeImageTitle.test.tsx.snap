// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoungeImageTitle /> renders LoungeImageTitle 1`] = `
.emotion-0 {
  background: #fff;
}

.emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  gap: 1.5rem;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.emotion-4 {
  margin: 0;
}

.emotion-6 {
  position: relative;
}

.emotion-8 {
  font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  display: block;
  width: 100%;
  height: 100%;
  border: 0;
  border-radius: 0.25rem;
}

.emotion-10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  gap: 1rem;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  className="emotion-0"
>
  <div
    className="emotion-1"
  >
    <div
      className="emotion-2 emotion-3"
      style={
        {
          "width": "7.5rem",
        }
      }
    >
      <figure
        className="emotion-4 emotion-5"
      >
        <div
          className="emotion-6 emotion-7"
        >
          <img
            alt="LoungeX"
            className="emotion-8 emotion-9"
            onError={[Function]}
            src="image.png"
            style={
              {
                "height": "5rem",
                "objectFit": "cover",
                "width": "7.5rem",
              }
            }
          />
        </div>
      </figure>
    </div>
    <div
      className="emotion-10"
    >
      <div
        className="emotion-11"
      >
        <h2
          style={
            {
              "fontSize": "12px",
              "margin": "0",
              "padding": "0",
            }
          }
        >
          LoungeX
        </h2>
        <strong>
          Athens, Greece
        </strong>
        <strong>
          80
        </strong>
      </div>
    </div>
  </div>
</div>
`;
